pkg/annotation/k8s.go

```diff
@@ -193,13 +196,13 @@ const (
 	LBIPAMSharingKeyAlias             = Prefix + "/lb-ipam-sharing-key"
 	LBIPAMSharingAcrossNamespace      = LBIPAMPrefix + "/sharing-cross-namespace"
 	LBIPAMSharingAcrossNamespaceAlias = Prefix + "/lb-ipam-sharing-cross-namespace"
-)
 
-var (
-	// CiliumPrefixRegex is a regex matching Cilium specific annotations.
-	CiliumPrefixRegex = regexp.MustCompile(`^([A-Za-z0-9]+\.)*cilium.io/`)
+	CECInjectCiliumFilters = CECPrefix + "/inject-cilium-filters"
```

Comment:
Is there a test case where this is used, rather than passing `true`/`false`?

Reply:
nope, will add one. Thanks!